lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -25,7 +25,7 @@
 
 """This package provides the TaurusGui class"""
 
-__all__ = ["DockWidgetPanel", "TaurusGui"]
+__all__ = ["DockWidgetPanel", "TaurusGui", "main"]
```

Comment:
Are you sure?
which generated launcher?
the `taurusgui` launcher already worked without this...